services/provider/server/server.go

```diff
@@ -1281,3 +1280,15 @@ func (s *OCSProviderServer) isConsumerMirrorEnabled(ctx context.Context, consume
 
 	return clientMappingConfig.Data[consumer.Status.Client.ID] != "", nil
 }
+
+func (s *OCSProviderServer) getMirroredClientID(ctx context.Context, consumer *ocsv1alpha1.StorageConsumer) (string, error) {
+	clientMappingConfig := &corev1.ConfigMap{}
+	clientMappingConfig.Name = util.StorageClientMappingConfigName
+	clientMappingConfig.Namespace = s.namespace
+
+	if err := s.client.Get(ctx, client.ObjectKeyFromObject(clientMappingConfig), clientMappingConfig); err != nil {
+		return "", client.IgnoreNotFound(err)
+	}
+
+	return clientMappingConfig.Data[consumer.Status.Client.ID], nil
```

Comment:
What if the key or the value does not exist? are we still going to return an empty value? and if so who is going to handle such a case?